config/settings.yml

```diff
@@ -57,6 +57,7 @@ environment:
   label: "Beta"
   selector_name: "beta"
 current_cycle: 2021
+allocation_cycle_year: 2022
```

Comment:
Shouldn't we keep this as `2021` for now until allocations open? Then we can deploy ahead of time.

For testing, we can set it to `2022` and `open` allocations in QA.

Reply:
Changed it to 2021